swing/src/net/sf/openrocket/gui/configdialog/FinSetConfig.java

```diff
@@ -228,31 +228,29 @@ private JPanel finTabPanel() {
 			@Override
 			public void actionPerformed(ActionEvent e) {
 				log.info(Markers.USER_MARKER, "Computing " + component.getComponentName() + " tab height.");
-				
+
+				double inRad = 0.0;
 				RocketComponent parent = component.getParent();
 				if (parent instanceof Coaxial) {
 					try {
 						document.startUndo("Compute fin tabs");
 						
 						List<CenteringRing> rings = new ArrayList<>();
-                        //Do deep recursive iteration
+                        // Do deep recursive iteration to find centering rings and determine
```

Comment:
nit: it'd be nice if the spacing were consistent. You've kept it consistent to the surrounding style, so not your fault. It's just that there's a mixture of tabs and spaces. Not necessary to fix, but preferred. Just spotted when reviewing.